dev/cognitiveservices/data-plane/Language/readme.md

```diff
@@ -34,6 +34,20 @@ modelerfour:
   lenient-model-deduplication: true
 ```
 
+### Release 2021-10-01
```

Comment:
I wouldn't add this version. It already shipped and is immutable by design. You'll just add these files to a new version after you update the api-version later.

Reply:
Removed it and added it under 2022-07-02-Preview